tarsier/tag_utils.ts

```diff
@@ -489,6 +514,19 @@ function insertTags(
     } else if (isImageElement(el)) {
       // Handle image elements
       const idSpan = create_tagged_span(idNum, el);
+      const idSymbol = extractIdSymbol(idSpan.textContent || "");
+      const idString = idSymbol ? `[ ${idSymbol} ${idNum} ]` : `[ ${idNum} ]`;
```

Comment:
we should not be manually reconstructing this anywhere. If we change how tags look, no we need to remember to also edit this area.

Reply:
Possible solution in above comment